src/main/java/frc/robot/RobotContainer.java

```diff
@@ -132,6 +151,10 @@ public void onExitDisabled() {
     signalingSubsystem.clearLEDs();
   }
 
+  private Command autonTest() {
+    return new InstantCommand(() -> SmartDashboard.putBoolean("NamedCommand test", true));
+  }
```

Comment:
The WPILib project template includes command/Autos.java utility class. I think that's where we'll want to put Auto command factories like this.

Reply:
Agreed, although I don't see many future scenarios where we have auto-only commands